client/angular/services/columnAssignmentService.js

```diff
@@ -6,13 +6,47 @@
     columnAssignmentService.$inject = ["tweetInfoService"];
 
     function columnAssignmentService(tweetInfoService) {
+
+        var store = {};
+
         return {
             ColumnData: ColumnData,
+            assignDisplayColumns: assignDisplayColumns,
+            clearStore: clearStore,
             assignColumns: assignColumns,
```

Comment:
None of them do strictly speaking, it might be better to have them sealed off - I figured there was no harm in leaving them open, since they haven't changed and can still be used as normal (although there's not much need to)